java/gandiva/src/main/java/org/apache/arrow/gandiva/evaluator/Projector.java

```diff
@@ -187,6 +187,45 @@ public static Projector make(Schema schema, List<ExpressionTree> exprs,
   public static Projector make(Schema schema, List<ExpressionTree> exprs,
       SelectionVectorType selectionVectorType,
       long configurationId) throws GandivaException {
```

Comment:
Can you add some java unit test..only with no-op sec cache..just to verify the jni flow